### PR TITLE
[GraphBolt][CUDA] refine the node_classification examples.

### PR DESCRIPTION
**Pull request by mfbalin**
## Description
Fixing minor typos and aligning the multiGPU example with the tutorial updated in #7126.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by mfbalin** [comment]
@TristonC you can run the papers100M with the updated multiGPU script in this PR.

File: examples/sampling/graphbolt/node_classification.py [hunk 1]
```diff
@@ -365,8 +365,9 @@ def parse_args():
         "--dataset",
         type=str,
         default="ogbn-products",
+        choices=["ogbn-arxiv", "ogbn-products", "ogbn-papers100M"],
         help="The dataset we can use for node classification example. Currently"
-        "dataset ogbn-products, ogbn-arxiv, ogbn-papers100M is supported.",
+        "ogbn-products, ogbn-arxiv, ogbn-papers100M datasets are supported.",
```
Comment (Skeleton003) [inline examples/sampling/graphbolt/node_classification.py:370]:
```suggestion
        " ogbn-products, ogbn-arxiv, ogbn-papers100M datasets are supported.",
```

**Comment by frozenbugs** [comment]
LGTM @RamonZhou please take a second look and approve.

File: examples/sampling/graphbolt/node_classification.py [hunk 2]
```diff
@@ -365,8 +365,9 @@ def parse_args():
         "--dataset",
         type=str,
         default="ogbn-products",
+        choices=["ogbn-arxiv", "ogbn-products", "ogbn-papers100M"],
```
Comment (Rhett-Ying) [inline examples/sampling/graphbolt/node_classification.py:368]: @caojy1998 FYI

**Review by RamonZhou — APPROVED** [review]
LGTM

**Pull request merged**